.github/workflows/helm.yml

```diff
@@ -183,7 +183,7 @@ jobs:
     name: Publish
     runs-on: ubuntu-latest
     # Allowing manual workflow_dispatch on main branches as version is always verified
-    if: (${{ inputs.publish-alpha == 'true' }} && github.event_name == 'workflow_dispatch') || (github.ref == 'refs/heads/main' && (github.event_name == 'push' || github.event_name == 'workflow_dispatch'))
+    if: ('${{ inputs.publish-alpha }}' == 'true' && github.event_name == 'workflow_dispatch') || (github.ref == 'refs/heads/main' && (github.event_name == 'push' || github.event_name == 'workflow_dispatch'))
```

Comment:
I think this part is not working as expected so it should be removed
```suggestion
```

---

What might be added tho is an additional condition excluding triggers from PR but I'm ok to merge as is

Reply:
That is what we wanted to do with the `github.event_name == 'workflow_dispatch'` but it seems a bit more complicated to identify where it was launched from a sub workflow :/